icp-config.yaml

```diff
@@ -1,5 +1,10 @@
 #kubelet_extra_args:
 kibana_install: true
-docker_extra_args:
```

Comment:
Why are we removing the docker_extra_args here?

Reply:
@gangchen03 can you assist answering the above?

Reply:
In my test, I don't see this is must for Ubuntu with docker version upgraded to 18.x. But we may need to keep it in there, just add them back as default:
docker_extra_args:
```
- --storage-driver=devicemapper
- --storage-opt dm.directlvm_device=/dev/sdb
```